backend/src/main.cpp

```diff
@@ -258,6 +258,29 @@ void SetCid(const Napi::CallbackInfo& info)
     UserSession::cid = cid;
 }
 
+void SetRadioEffects(const Napi::CallbackInfo& info)
+{
+    auto radioEffects = info[0].As<Napi::String>().Utf8Value();
+    auto enableInputFilters = false;
+    auto enableOutputEffects = false;
+
+    if (radioEffects == "off") {
+        enableInputFilters = true;
+        enableOutputEffects = true;
+    }
```

Comment:
Shouldn't this be `false` instead of `true` if the goal is to disable the effects?

Reply:
Oh god. Typical copy and paste error where I forgot to change "off" to "on" in this case.